test/sds/SDS_Regression.ipynb

```diff
@@ -878,6 +891,100 @@
     "    print(f'HOSS is not configured for environment: \"{harmony_environment}\" - skipping test.')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "751e3ca8",
+   "metadata": {},
+   "source": [
+    "### HOSS temporal subset request:\n",
+    "\n",
+    "This request will combine a variable and temporal subset for a granule in the M2T1NXSLV collection (MERRA-2). The result will include the requested variable and the three associated dimension variables:\n",
+    "\n",
+    "* `/H1000`\n",
+    "* `/lat`\n",
+    "* `/lon`\n",
+    "* `/time`\n",
+    "\n",
+    "Furthermore, the temporal dimension and the science variable (`/H1000`) will be limited to the specified temporal range. MERRA-2 is gridded at hourly intervals, so the 4-hour time range (12pm - 4pm on 9th June 2021) will return 4 time values.\n",
+    "\n",
+    "For the granule being tested, all time values are expressed as minutes since 2021-06-09T00:30:00Z, and each pixel spans 30 minutes in either direction from the stated value (cells are centre-aligned). As such:\n",
+    "\n",
+    "* 12pm is the leading edge of the cell centred at 12:30pm, which is 720 minutes since 00:30am.\n",
```

Comment:
```suggestion
    "* 12pm is the leading edge of the cell centered at 12:30pm, which is 720 minutes since 00:30am.\n",
```

Reply:
I think I'm going to leave my UK English spelling as-is there...

Reply:
... I ended up swapping out both these instances for "with a centre value of".

Reply:
... I ended up swapping both of these instances for "...with a centre value of...".